### PR TITLE
[ci skip] Fix typos

### PR DESCRIPTION
**Pull request by yui-knk**

### CODE REVIEW

File: ext/bigdecimal/bigdecimal.h [hunk 1]
```diff
@@ -209,11 +209,11 @@ extern VALUE rb_cBigDecimal;
...
     VALUE  obj;     /* Back pointer(VALUE) for Ruby object.     */
     size_t MaxPrec; /* Maximum precision size                   */
-                    /* This is the actual size of pfrac[]       */
+                    /* This is the actual size of frac[]        */
                     /*(frac[0] to frac[MaxPrec] are available). */
     size_t Prec;    /* Current precision size.                  */
-                    /* This indicates how much the.             */
-                    /* the array frac[] is actually used.       */
+                    /* This indicates how much the              */
+                    /* array frac[] is actually used.           */
```
Comment (zzak) [inline ext/bigdecimal/bigdecimal.h]: :space_invader:

Comment (yui-knk) [inline ext/bigdecimal/bigdecimal.h]: @zzak Is something wrong?

Comment (zzak) [inline ext/bigdecimal/bigdecimal.h]: You added whitespace to lines 215 and 216, maybe mistake?

Comment (yui-knk) [inline ext/bigdecimal/bigdecimal.h]: I added intentionally to aline end of comments (`*/`)

Comment (zzak) [inline ext/bigdecimal/bigdecimal.h]: なるほど

Comment (yui-knk) [inline ext/bigdecimal/bigdecimal.h]: 😄

**Comment by zzak** [comment]
あざっす！

**Comment by yui-knk** [comment]
👍

**Pull request merged**